starport/pkg/spn/spn.go

```diff
@@ -478,86 +480,133 @@ func (c *Client) ProposalGet(ctx context.Context, accountName, chainID string, i
 	return c.toProposal(*res.Proposal)
 }
 
-// ProposeAddAccount proposes to add a validator to chain.
-func (c *Client) ProposeAddAccount(ctx context.Context, accountName, chainID string, account ProposalAddAccount) error {
-	clientCtx, err := c.buildClientCtx(accountName)
-	if err != nil {
-		return err
-	}
+// Proposition configures Proposal to make a proposition.
+type Proposition func(*Proposal)
 
-	addr, err := types.AccAddressFromBech32(account.Address)
-	if err != nil {
-		return err
+// AddAccountProposition creates an add account proposition.
```

Comment:
What is the difference between proposition and proposal?

Reply:
nothing other than just naming. I felt proposition word is better while proposing something because it is also a verb (describes an action).

Reply:
Also not sure about "proposition". Since we're using a "proposal" on `spn`, shouldn't we stick to the same terminology?

Reply:
I had to use some name other than _Proposal_ since we already have a type with the same name. But that's fine I can add _Option_ suffix in the end to avoid conflicts.

Reply:
@ilgooz oh, I didn't realize that 🙂 sounds good 👍

Reply:
solved by 44d2c26.